app/views/content/urgent-call-for-qualified-teachers/_faq.html.erb

```diff
@@ -330,12 +330,13 @@
     </p>
 
     <p>
-      If you’re thinking about returning to teach modern foreign languages, maths or physics
-      at secondary school in England, you can <a href="/tta-service">get a return to teaching adviser</a> who can give
-      you free one-to-one support on applying for roles, writing personal statements and
-      preparing for interviews. They can also help you find courses to enhance your subject
-      knowledge.
-    </p>
+      If you’re thinking about returning to teach at secondary school in England, you could
```

Comment:
you could be eligible for rather than could get? to emphasise that it is not available for everyone?